content/en/docs/reference/lifecycle-events/lifecycle_hooks/_index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Lifecycle Hooks
+content_type: reference
+weight: 10
+---
+
+# Overview of lifecycle hooks
```

Comment:
This has a lot of overlap with https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/

I think we should leave the overview and prose to the existing page, and focus on the specific state transitions that are being tested here.